app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
   <%#= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
   <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css"/>
+  <link rel="icon" type="image/png" href="https://platforma-slovensko-digital-uploads.s3.dualstack.eu-central-1.amazonaws.com/optimized/2X/e/ee54258f8b47fabe977dc40ded15a4ae78e7e044_2_32x32.png">
```

Comment:
co je toto za URL? nebude to nahodou iba cache?

Reply:
Som zobral z https://platforma.slovensko.digital/, myslim ze to je Amazon AWS cloud z ktoreho to taha co je na to zriadeny

Reply:
Je to cache? Mne to pride ako niaky vygenerovany fix link na graficky asset

Reply:
ale je to na tej adrese vzdy dostupne? nezmeni sa niekedy ten fingerprint? (redeploy atd.)

Reply:
Yop, to som sa planoval spytat Jana v Po tiez.

Reply:
@smith558 ja by som odporucal ikonu stiahnut a servirovat normalne cez asset pipeline, e.g. https://medium.com/@josephcardillo/how-to-add-a-favicon-to-your-rails-app-9676336f7006